lib/driver.rb

```diff
@@ -0,0 +1,75 @@
+require_relative "csv_record"
+
+module RideShare
+  class Driver < RideShare::CsvRecord
+    attr_reader :id, :name, :vin, :status, :trips
+
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: [])
+      super(id)
+
+      @name = name
+      @vin = validate_vin(vin)
+      @status = validate_status(status)
+      @trips = trips
+    end
+
+    def add_trip(trip)
+      @trips << trip
+    end
+
+    def change_status
```

Comment:
Love that this is a method on Driver rather than writing this code in TripDispatcher!